### DISCUSSION TITLE
Run kubectl command get error

### DISCUSSION THREAD

**Discussion by jonastian** [kubernetes-client/csharp#1184] [Q&A] [answered]
I use .net6, start minikube in Hyper-v locally, and I can run the following commands in my local bash
$ kubectl get kafka --namespace debezium-example

When I use KubernetesClient, run the following code and get the error: **_bash: kubectl: command not found_**

               await client.NamespacedPodExecAsync(pod.Metadata.Name, @namespace, container.Name, new[] { "bash","-c", "kubectl get kafka --namespace debezium-example" }, false,
                                 handler,
                                 new CancellationToken());

Could help me with this?

**Comment by tg123** [comment, marked as answer]
you should make sure the kubectl binary is inside your pod

**Reply by jonastian** [reply]
thank you!
